dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNFictBusInterfaceIIDM.cpp

```diff
@@ -0,0 +1,110 @@
+//
+// Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+// This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+// simulation tool for power systems.
+//
+
+/**
+ * @file  DYNBusInterfaceIIDM.cpp
+ *
+ * @brief Bus data interface  : implementation file for IIDM interface
+ *
+ */
+#include <cmath>  // M_PI
+
+#include <powsybl/iidm/Bus.hpp>
+#include <powsybl/iidm/VoltageLevel.hpp>
+#include <powsybl/iidm/BusbarSection.hpp>
+#include <powsybl/iidm/Connectable.hpp>
```

Comment:
I guess the powsybl includes are using are now useless

Reply:
yes, done